pyproject.toml

```diff
@@ -12,8 +12,6 @@ readme = "README.md"
 license = {text = "MIT"}
 dependencies = [
     "stripe",
-    "httpx-oauth",
-    "pyjwt",
-    "streamlit>=1.30.0",
+    "streamlit-nightly",
```

Comment:
Is it worth just limiting this to streamlit >= whenever auth was added, so that:
1. People using this who already have streamlit don't need an extra dependency when they install this
2. You don't end up getting some weird/broken stuff that won't be there in the released version of streamlit

Reply:
yeah I was going to switch to 1.42 ++